pkg/sqlite/scene_filter.go

```diff
@@ -455,12 +455,12 @@ func (qb *sceneFilterHandler) performerFavoriteCriterionHandler(performerfavorit
 			if *performerfavorite {
 				// contains at least one favorite
 				f.addLeftJoin("performers", "", "performers.id = performers_scenes.performer_id")
-				f.addWhere("performers.favorite = 1")
+				f.addWhere(fmt.Sprintf("performers.favorite = %s", getDBBoolean(true)))
```

Comment:
Just as a nit, that using `fmt.Sprintf` for string concat is really bad for perf, it causes extra allocations and pressure on the GC
```suggestion
				f.addWhere("performers.favorite = " + getDBBoolean(true))
```